packages/app/src/client/views/integrations/all/Show.tsx

```diff
@@ -32,10 +32,5 @@ export const ShowIntegration = () => {
 
   if (!integrationDef) return <NotFound />;
 
-  return (
-    <integrationDef.Show
-      integration={integration}
-      integrationDef={integrationDef}
-    />
-  );
+  return <integrationDef.Show />
```

Comment:
The now working type checking indicated that these props are in fact not needed.